modules/edu.gemini.seqexec.model/src/main/scala/edu/gemini/seqexec/model/dhs.scala

```diff
@@ -3,6 +3,6 @@ package edu.gemini.seqexec.model
 object dhs {
 
   // Common data types
-  type ObsId = String
+  type ImageFileId = String
```

Comment:
I renamed the type for image file identifier, because the previous name was getting confusing, as the ODB uses the term "Observation Id" to refer to the identification for a sequence.